manifests/0000_12_etcd-operator_08_etcdquorumguard_deployment.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: etcd-quorum-guard
+  namespace: openshift-etcd-operator
+  annotations:
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+spec:
+  replicas: 3
```

Comment:
Could the operator control the amount of replicas? In OKD's fork we used https://github.com/openshift/machine-config-operator/commit/ddc89ca7814c38c88cfc0c27d083f82f899b5b85 in MCO - when `useUnsupportedUnsafeNonHANonProductionUnstableEtcd` is enabled etcd-quorum-guard is scaled back to 1

Reply:
Operator controlling this Deployment so it can do intelligent things makes sense to me.  But can that happen after the pivot from machine-config operator to etcd operator?  We certainly don't want the etcd operator trying to do intelligent Deployment management while the machine-config operator is still trying to hand this Deployment off to the cluster-version operator, because the CVO and etcd operator would be fighting each other.

Reply:
There should be any issue during transition from MCO to CEO as these are different deployments in different namespaces

Reply:
> There should be any issue during transition from MCO to CEO as these are different deployments in different namespaces

Oh.  Why is the quorum guard not living in the same namespace as etcd?  Regardless, I'd still like the cross-repo pivot to be as boring and conservative as possible.  That way, if we flub the transition to intelligent etcd-operator management, we can easily revert the change without having to involve the MCO.  I don't think it's wrong to attempt the bigger pivot in a single PR, if folks are feeling more aggressive.  But is there much upside to moving to etcd-operator management in a single PR?

Reply:
>Why is the quorum guard not living in the same namespace as etcd?

That would make MCO watch events in different namespace (currently etcd-quorum-guard lives in MCO namespace). 

>  I don't think it's wrong to attempt the bigger pivot in a single PR, if folks are feeling more aggressive

Sure, I don't mind splitting replica control in a separate PR